### PR TITLE
fix: for timeline.gotoAndStop() goes to, but doesn't stop

### PR DESCRIPTION
**Pull request by stristr**
OK to merge.

Short review.

Purpose of changes:

- Fix for [timeline.gotoAndStop() goes to, but doesn't stop](https://app.asana.com/0/539750334128224/528728886739983/f)

Summary of work (include Asana links if any):

- [x] Fix bug and write test.

Completed checkin tasks:

- [x] Did manual testing of interrelated functionality
- [x] Ran `$ yarn lint-all` and fixed all formatting issues
- [x] Ran `$ yarn test-all` and all tests passed
- [x] Wrote an automated test covering new functionality

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -7330,6 +7326,10 @@ loose-envify@^1.0.0, loose-envify@^1.1.0, loose-envify@^1.2.0, loose-envify@^1.3
   dependencies:
     js-tokens "^3.0.0"
 
+lottie-web@5.1.7:
+  version "5.1.7"
+  resolved "https://registry.yarnpkg.com/lottie-web/-/lottie-web-5.1.7.tgz#5a4c9163209af12333637282b02032782fbfc530"
+
 loud-rejection@^1.0.0:
```
Comment (stristr) [inline yarn.lock:7333]: Unrelated (forgot to commit this previously).

Comment (roperzh) [reply yarn.lock:7333]: ~I find really annoying that GH not only was hiding the diff (which is sort of ok) but also your comment.~

edit: latency...

**Comment by roperzh** [comment]
Added Matthew as a reviewer since he's more familiar with this.

**Comment by tsortwehttam** [comment]
Code LGTM. 🥇 for the test. Merging.

**Pull request merged**